scripts/scr_setinventory.gml

```diff
@@ -0,0 +1,31 @@
+//Destroy inventory tiara object if it exists in room
```

Comment:
This whole file should probably be broken down into multiple smaller files, too many things being achieved at once for a single function call.
Consider:
`scr_inventory_set_tiara(bool)`
`scr_inventory_set_coat(bool)`
The excessive use of create/destroy is really hindering this one. Possibly revisit your architecture...